src/logics/LALogic.cc

```diff
@@ -837,13 +841,15 @@ void SimplifyConstTimes::constSimplify(const SymRef& s, const vec<PTRef>& terms,
         terms_new.push(con);
     else
     {
-        Pterm& p = l.getPterm(plus);
```

Comment:
Maybe make explicit that here con != PTRef_Undef && plus != PTRef_Undef with a comment, like moving the above comment "distribute the constant over the first sum" here?

Reply:
Added assertion to make it explicit. Also, moved the comment.